datafusion/core/src/physical_plan/filter.rs

```diff
@@ -133,8 +134,8 @@ impl ExecutionPlan for FilterExec {
         partition: usize,
         context: Arc<TaskContext>,
     ) -> Result<SendableRecordBatchStream> {
+        debug!("Start FilterExec::execute for partition: {}", partition);
```

Comment:
I wonder if it would be useful to log `context` as well (perhaps `session_id` and `task_id`)?

https://github.com/apache/arrow-datafusion/blob/7a6317a0e31829ab06669a57987272b97e4694f4/datafusion/core/src/execution/context.rs#L1430

This comment applies to this entire PR